server/functions.lua

```diff
@@ -8,12 +8,19 @@ QBCore.UsableItems = {}
 -- ex: local player = QBCore.Functions.GetPlayer(source)
 -- ex: local example = player.Functions.functionname(parameter)
 
+---Gets the coordinates of an entity
```

Comment:
Do these comments need to have `---` or `--`?

See: https://github.com/lunarmodules/LDoc